src/components/SwapForm/hooks/useGetRoute.ts

```diff
@@ -18,6 +19,8 @@ export type ArgsGetRoute = {
   parsedAmount: CurrencyAmount<Currency> | undefined
   currencyIn: Currency | undefined
   currencyOut: Currency | undefined
+
+  // TODO: remove this
```

Comment:
TODO?

Reply:
Will do in the next PR. To remove referral program completely